collections/ansible_collections/rhsiqe/skupper/roles/run_curl/tests/test_playbook.yml

```diff
@@ -23,78 +9,10 @@
       ansible.builtin.include_role:
         name: rhsiqe.skupper.generate_namespaces
 
-    - name: Installing Skupper
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.install_skupper
-      when:
-        - not skip_skupper_install | default(false) | bool
-      vars:
-        install_skupper_install_output_path: "{{ temp_dir_path }}"
-
-    - name: Installing Skupper Controller
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.install_skupper_controller
-      when:
-        - not skip_skupper_controller_install | default(false) | bool
-
-    - name: Deploying backend and frontend
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.deploy_workload
-      vars:
-        deploy_workload_output_path: "{{ temp_dir_path }}"
-
-    - name: Creating Skupper sites
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.skupper_site
-      vars:
-        skupper_site_output_path: "{{ temp_dir_path }}"
-
-    - name: Exposing the connector at the backend at east site
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.expose_connector
-      when:
-        - "'east' in inventory_hostname"
-      vars:
-        expose_connector_output_path: "{{ temp_dir_path }}"
-
-    - name: Consuming the service at the frontend at west site
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.consume_service
-      when:
-        - "'west' in inventory_hostname"
-      vars:
-        consume_service_output_path: "{{ temp_dir_path }}"
-
-    - name: Granting access to the frontend at west site
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.access_grant
-      when:
-        - "'west' in inventory_hostname"
-      vars:
-        access_grant_output_path: "{{ temp_dir_path }}"
-
-    - name: Exposing the service at the frontend at west site
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.expose_service
-      when:
-        - "'west' in inventory_hostname"
-
-    - name: Linking east site to west site
-      ansible.builtin.include_role:
-        name: rhsiqe.skupper.link_site
-      when:
-        - "'east' in inventory_hostname"
-
```

Comment:
I don't understand where all those instructions go.  They went into generate_namespace or run_curl?

Reply:
Ah, this is  just testing curl, so that was all removed as unnecessary.  Gotcha.